tools/sift.go

```diff
@@ -333,6 +339,7 @@ var FindErrorPatternLogs = mcpgrafana.MustTool(
 	"Searches Loki logs for elevated error patterns compared to the last day's average, waits for the analysis to complete, and returns the results including any patterns found.",
 	findErrorPatternLogs,
 	mcp.WithTitleAnnotation("Find error patterns in logs"),
+	mcp.WithReadOnlyHintAnnotation(true),
```

Comment:
do we need to add "false" on the Idempotent hint annotation? Not sure if that really affects performance

Reply:
Yeah good point actually, I'm not sure if the default is false or just 'assume nothing'...

Reply:
The [docs](https://modelcontextprotocol.io/docs/concepts/tools#available-tool-annotations) indicate that the default is `false` so I don't think we need to add the idempotent one.